src/ProjectService.Models.Dto/Enums/ProjectStatusType.cs

```diff
@@ -1,10 +1,12 @@
 ﻿namespace LT.DigitalOffice.ProjectService.Models.Dto.Enums
 {
-    public enum ProjectClosedReason
+    public enum ProjectStatusType
```

Comment:
I still think. that we need only 3 types, but it is not critical

Reply:
yes, not critical